buffer-api/src/main/java/io/netty/buffer/api/MemoryManagers.java

```diff
@@ -72,4 +99,12 @@ static MemoryManagers getManagers() {
      * @return An off-heap {@link MemoryManager}.
      */
     MemoryManager getNativeMemoryManager();
+
+    /**
+     * Get the name for this implementation, which can be used for finding this particular implementation via the
+     * {@link #lookupImplementation(String)} method.
+     *
+     * @return The name of this memory managers implementation.
+     */
+    String implementationName();
```

Comment:
nit: I wonder if it would be better to be more consistent. Sometimes methods are prefixed with `get` and sometimes not.